internal/codewriter/codewriter.go

```diff
@@ -99,7 +99,7 @@ func (c *Writer) write(w io.Writer) {
 	fmt.Fprintf(w, "// Code generated by zero. DO NOT EDIT.\n")
 	fmt.Fprintf(w, "package %s\n", c.pkg)
 	fmt.Fprintf(w, "\n")
-	imports := []string{}
+	imports := []string{"github.com/alecthomas/zero"}
```

Comment:
Ah this should be put into the generator package!

Reply:
It should be imported at the point of use, so we make sure we only import exactly what we use. I must have missed a spot!

Reply:
ah gotcha! found it thanks mang